### PR TITLE
[lab-13] support x2y2

### PR DESCRIPTION
**Pull request by viet10101988**
because requesting from browser using sdk is not allowed by x2y2, we will have to request via backend

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@refinableco/refinable-sdk",
-  "version": "4.1.0-next.2",
+  "version": "4.1.1-next.0",
```
Comment (maximgeerinck) [inline package.json:3]: is this patch version needed? can as well be `.3` no?

**Review by maximgeerinck — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (viet10101988) [reply package.json:3]: I was unsure, it adds a minor feature so I should increase to 4.1.1

Comment (maximgeerinck) [reply package.json:3]: that's a patch
let's put `4.1.0-next.3`

Comment (viet10101988) [reply package.json:3]: ok

**Pull request merged**